Mathlib/Analysis/Normed/Ring/Basic.lean

```diff
@@ -370,6 +370,16 @@ theorem Finset.nnnorm_prod_le {α : Type*} [NormedCommRing α] [NormOneClass α]
     (f : ι → α) : ‖∏ i ∈ s, f i‖₊ ≤ ∏ i ∈ s, ‖f i‖₊ :=
   (s.norm_prod_le f).trans_eq <| by simp [NNReal.coe_prod]
 
+lemma norm_natAbs (z : ℤ) :
+    ‖(z.natAbs : α)‖ = ‖(z : α)‖ := by
```

Comment:
Should that be simp? Same below

Reply:
Because `‖↑z.natAbs‖ = ‖↑z‖` becomes `‖↑|z|‖ = ‖↑z‖` in any ring.